lib/task_helpers/exports/customization_templates.rb

```diff
@@ -1,7 +1,7 @@
 module TaskHelpers
   class Exports
     class CustomizationTemplates
-      EXCLUDE_ATTRS = %i(created_at updated_at id pxe_image_type_id class).freeze
+      EXCLUDE_ATTRS = %i(created_at updated_at pxe_image_type_id).freeze
```

Comment:
This change makes me think that this maybe isn't the right way to do this.

Reply:
Could you elaborate your thoughts? Whats the point in removing these attributes anyway? You can still throw them away while importing.